x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/factories/utils/flatten.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SearchTypes } from '../../../../../../common/detection_engine/types';
+
+export const flatten = (
```

Comment:
we could call this `flattenWithPrefix` as the prefix seems to be an important part of this function

Reply:
Thanks @ecezalp , good suggestion